app/lib/features/events/pages/create_event_page.dart

```diff
@@ -477,4 +481,15 @@ class CreateEventPageConsumerState extends ConsumerState<CreateEventPage> {
       );
     }
   }
+
+  Future<String?> selectSpace() async {
+    final newSelectedSpaceId = await selectSpaceDrawer(
+      context: context,
+      currentSpaceId: ref.read(selectedSpaceIdProvider),
+      canCheck: 'CanPostPin',
```

Comment:
and that helper should have an explicit `canCheck`-param ... as this copy-paste lead to using the wrong key ;) 

```suggestion
      canCheck: 'CanPostEvent',
```

Reply:
Opss! Thanks for noticing.